apps/web/src/app/(content)/changelog/page.tsx

```diff
@@ -26,10 +27,19 @@ export const metadata: Metadata = {
   },
 };
 
-export default async function Changelog() {
-  const changelogs = allChangelogs.sort(
+export default function ChangelogClient(context: Context) {
+  const currentPage = parseInt(context.searchParams.page) || 1
```

Comment:
Please use zod validation here. An example:

https://github.com/openstatusHQ/openstatus/blob/main/apps/web/src/app/app/%5BworkspaceSlug%5D/(dashboard)/monitors/(overview)/page.tsx#L20

(same for Blog page)